### PR TITLE
Ft/use logger for shell

### PR DESCRIPTION
**Pull request by ThibaultRiviere**

### CODE REVIEW

**Comment by MathieuCassagne** [comment]
:+1:

**Comment by AntoninCoulibaly** [comment]
:+1:

File: bin/shell.js [hunk 1]
```diff
@@ -0,0 +1 @@
+require('../lib/shell.js')();
```
Comment (michael-zapata) [inline bin/shell.js]: How about making it executable instead, using `!#/usr/bin/env node` to start it?

**Comment by ThibaultRiviere** [comment]
Yes why not

**Comment by michael-zapata** [comment]
Small nit, you could use it in the "bin" section of the `package.json`

**Comment by ThibaultRiviere** [comment]
done :)

**Pull request merged**